projects/firmware_103/src/main.c

```diff
@@ -0,0 +1,74 @@
+#include "../../../libraries/ms-common/inc/adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+typedef enum { B2_BUTTON = 0, NUM_BUTTONS } Button;
+
+typedef enum { A6_SENSOR = 0, NUM_SENSORS } Sensor;
+
+static GpioAddress button_addresses[] = {
+  [B2_BUTTON] = { .port = GPIO_PORT_B, .pin = 2 },

```

Comment:
Add comma after .pin=2

Reply:
You also probably don't need to make this an array. I like that it makes it scaleable, but in this case we don't really need it.